qiskit_optimization/__init__.py

```diff
@@ -26,16 +26,17 @@
 A uniform interface as well as automatic conversion between different problem representations
 allows users to solve problems using a large set of algorithms, from variational quantum algorithms,
 such as the Quantum Approximate Optimization Algorithm
-(:class:`~qiskit.algorithms.minimum_eigensolver.QAOA`), to
+(:class:`~qiskit_algorithms.QAOA`), to
 `Grover Adaptive Search <https://arxiv.org/abs/quant-ph/9607014>`_
 (:class:`~algorithms.GroverOptimizer`), leveraging
-fundamental :mod:`~qiskit.algorithms.minimum_eigensolver` provided by Qiskit Terra.
+fundamental :mod:`~qiskit_algorithms.minimum_eigensolvers` provided by
```

Comment:
With qiskit_algorithms this will not form a link as it will not resolve to any documentation since the minimum_eigensolvers module is not documented as it was before (before it was not a link either since it lacked the `s` at the end). I think its ok.

The Qiskit Algorithms link on the next line I think would be better to the published docs though so you go from docs to docs ie. use https://qiskit.org/ecosystem/algorithms/

And for the module above if you preferred it to be a link then it could be changed to make it a link to where the `Minimum Eigensolvers` are documented ie this https://qiskit.org/ecosystem/algorithms/apidocs/qiskit_algorithms.html#minimum-eigensolvers

Reply:
Thanks. I updated it to mention minimum eigensolvers with https://qiskit.org/ecosystem/algorithms/apidocs/qiskit_algorithms.html#minimum-eigensolvers instead of `~qiskit_algorithms.minimum_eigensolvers`

Reply:
Sorry. I forgot to push it. 10d057b